### PR TITLE
ci: run build image on self-hosted arm machine

### PR DESCRIPTION
**Pull request by YangKeao**
Signed-off-by: YangKeao <yangkeao@chunibyo.icu>

### What problem does this PR solve?

Run build image ci on self-hosted arm machine

### Related changes

- [ ] Need to update `chaos-mesh/website`
- [ ] Need to update `Dashboard UI`
- Need to **cheery-pick to release branches**
  - [x] release-2.2
  - [x] release-2.1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,12 +8,14 @@ For more information and how-to, see [RFC: Keep A Changelog](https://github.com/
 
 ## [Unreleased]
 
-- Nothing
-
```
Comment (STRRL) [inline CHANGELOG.md:12]: please revert these changes on CHANGELOG.

**Review by STRRL — COMMENTED** [review]
rest LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (YangKeao) [reply CHANGELOG.md]: Done.

**Comment by YangKeao** [comment]
https://github.com/chaos-mesh/chaos-mesh/runs/7227702369?check_suite_focus=true This action runs well :beers:

**Review by cwen0 — APPROVED** [review]
LGTM

**Comment by YangKeao** [comment]
PTAL @STRRL

**Review by STRRL — APPROVED** [review]
LGTM!
